blazor/dropdown-list/style.md

```diff
@@ -270,6 +282,64 @@ In the following code, a single list Item is hidden using jsinterop.
 
 ![Blazor DropDownList with Popup Width](./images/style/blazor_dropdown_disable-listitem.png)
 
+## Adding conditional HTML attribute to list item
+
+You can achieve adding attributes to the li items based on datasource value with the help of JSInterop. In the, Opened event need to call the client side script by passing the required arguments (datasource and id) and add the attributes based on the datasource value obtained from the server.
```

Comment:
This is achieved by adding attributes to the li items based on the data source value with the help of the JSInterop. In the opened event, call the client-side script by passing the required arguments (data source and id) and adding the attributes based on the data source value obtained from the server.